Terminal.Gui/Views/Scroll.cs

```diff
@@ -0,0 +1,355 @@
+﻿//
+// Scroll.cs: Vertical or horizontal scroll
+//
+// Author: BDisp
+//
+// Licensed under the MIT license
+//
+
+namespace Terminal.Gui;
+
+/// <summary>
+///     Represents the "inside part" of a scroll bar, minus the arrows.
+/// </summary>
+public class Scroll : View
+{
+    /// <inheritdoc/>
+    public Scroll ()
+    {
+        WantContinuousButtonPressed = true;
+        ClearOnVisibleFalse = false;
```

Comment:
Why is this needed?

I've never understood why this property is there. It seems like a hack to work around something that should be fixed at a deeper level.